src/pages/library/WorkflowsLibrary.tsx

```diff
@@ -145,11 +136,8 @@ const CuratedWorkflowsSection = () => {
 export const WorkflowsLibrary = () => {
   const dockstoreUrl = `${getConfig().dockstoreUrlRoot}/search?_type=workflow&descriptorType=WDL&searchMode=files`;
 
-  // Set to static `workflows` and remove feature flag
-  const workflowsRepoUrl: string = isFeaturePreviewEnabled(FIRECLOUD_UI_MIGRATION)
-    ? Nav.getLink('workflows')
-    : `${getConfig().firecloudUrlRoot}/?return=${getEnabledBrand().queryName}#methods`;
-  const workflowsRepoLogo = isFeaturePreviewEnabled(FIRECLOUD_UI_MIGRATION) ? terraLogo : broadLogo;
+  const workflowsRepoUrl: string = Nav.getLink('workflows');
```

Comment:
Out of scope for this PR so feel free to ignore/defer, but I noticed while testing this that TWR opens in a new tab when you click on it, which seems to run counter to the idea that it's a regular part of Terra now. It does NOT open in a new tab when you enter through the "add a workflow" flow in a workspace.

Reply:
Oh, interesting, @salonishah11 what do you think? I would love to release to GA tomorrow so it goes to prod by Friday. Happy to create a follow up ticket too.

Reply:
IMO links should behave according to the user's preference:

Click = navigate in same tab
Command/control + click = open new tab

Reply:
Chatted with Beth and team and definitely not a blocker for release, but worth a follow up ticket for sure

Reply:
Agreed. Ship it!

Reply:
Agreed that this should not block merging of this PR. Created a follow up [ticket](https://broadworkbench.atlassian.net/browse/AN-408).